net/acme-common/files/acme.init

```diff
@@ -140,26 +142,27 @@ load_globals() {
 
 start_service() {
 	mkdir -p $run_dir
-	exec 200>$run_dir/lock
-	if ! flock -n 200; then
-		log err "Another ACME instance is already running."
-		exit 1
-	fi
-
-	trap cleanup EXIT
+	mkdir -p "$CHALLENGE_DIR"
 
-	config_load acme
-	config_foreach load_globals acme
+	grep -q '/etc/init.d/acme' /etc/crontabs/root 2>/dev/null || {
+		echo "0 0 * * * /etc/init.d/acme renew" >>/etc/crontabs/root
+	}
+}
 
-	config_foreach get_cert cert
+stop_service() {
+	sed -i '\|/etc/init.d/acme|d' /etc/crontabs/root
 }
```

Comment:
Similarly, removing the cron entry on 'stop' may be surprising, so adding a `service_stopped` function that prints a message about this would be good.

Reply:
As mentioned above, this is only done when I explicitly call the `/etc/init.d/acme stop` function. This does not happen with a clean shutdown because the `/etc/init.d/acme` script does not have a `STOP=<number>` entry.

As it is an embedded system, you want to have as little write access as possible to the hard drive memory, which in most cases is a NAND.

Reply:
Right, but it's a change in behaviour, so I think a notice message is warranted...